File: CPP/Petshop.cpp
#include <iostream>
#include <string>

using namespace std;

class Petshop{
    private:
        string ID_produk;
        string nama_produk;
        string kategori;
        int harga;
    
    public:
        Petshop(){
            this->ID_produk = "";
            this->nama_produk = "";
            this->kategori = "-";
            this->harga = 0;
        }

        void setId(string id)
        {
            this->ID_produk = id;
        }

        void setNama(string nama)
        {
            this->nama_produk = nama;
        }

        void setKategori(string kategori)
        {
            this->kategori = kategori;
        }

        void setHarga(int harga)
        {
            this->harga = harga;
        }

        string getId()
        {
            return this->ID_produk;
        }

        string getNama()
        {
            return this->nama_produk;
        }

        string getKategori()
        {
            return this->kategori;
        }

        int getHarga()
        {
            return this->harga;
        }

        ~Petshop() {}
};

File: Python/Petshop.py
class Petshop:
    def __init__(self, ID_produk="", nama_produk="", kategori="-", harga=0):
        self.ID_produk = ID_produk
        self.nama_produk = nama_produk
        self.kategori = kategori
        self.harga = harga
    
    def setId(self, id):
        self.ID_produk = id
    
    def setNama(self, nama):
        self.nama_produk = nama
    
    def setKategori(self, kategori):
        self.kategori = kategori
    
    def setHarga(self, harga):
        self.harga = harga
    
    def getId(self):
        return self.ID_produk
    
    def getNama(self):
        return self.nama_produk
    
    def getKategori(self):
        return self.kategori
    
    def getHarga(self):
        return self.harga

